.github/workflows/publish.yml

```diff
@@ -4,8 +4,8 @@ env:
   JAVA_OPTS: "-Xms512m -Xmx6048m -Xss128m -XX:ReservedCodeCacheSize=512m -server"
   GRADLE_OPTS: "-Xms512m -Xmx6048m -Xss128m -XX:ReservedCodeCacheSize=512m -server"
   TERM: xterm-256color
-  SONATYPE_USER: ${{ secrets.SONATYPE_USER }}
```

Comment:
I updated the setup so REPOSITORY_PWD and REPOSITORY_USER are now available to the build. You can make the change here as well.

Reply:
OK I made the change and rebased on latest changes. Is CI broken for some reason related to the apereo org?

Reply:
Seemed to be an issue with GH actions. Not sure. We should be back on track.